src/components/Select.js

```diff
@@ -41,7 +42,8 @@ class Select extends React.Component {
     selected: optionShape,
     selectedStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
     theme: themeShape,
-    valid: PropTypes.bool
+    valid: PropTypes.bool,
+    withSearch: PropTypes.bool,
```

Comment:
Can you add a default state of false for this in `defaultProps`. I know it technically doesn't matter but it's nice to have some clarity around prop states for stuff like this.

Reply:
You'll also need to make sure you update the associated docs with this new prop as well as the change log in the docs directory.

Reply:
Absolutely!